certification/artifacts/artifacts.go

```diff
@@ -20,59 +25,40 @@ const DefaultArtifactsDir = "artifacts"
 
 func init() {
 	// set the singleton to the default value.
-	ads = DefaultArtifactsDir
+	cwd, _ := os.Getwd()
+	ads = filepath.Join(cwd, DefaultArtifactsDir)
 }
 
 // SetDir sets the package level artifacts directory. This
 // can be a relative path or a full path.
 func SetDir(s string) {
-	ads = s
+	fullPath := s
+	if !strings.HasPrefix(s, "/") {
+		cwd, _ := os.Getwd()
+		fullPath = filepath.Join(cwd, s)
+	}
+	ads = fullPath
 }
 
 // Reset restores the default value for the Artifacts Directory.
 func Reset() {
-	ads = DefaultArtifactsDir
+	cwd, _ := os.Getwd()
+	ads = filepath.Join(cwd, DefaultArtifactsDir)
 }
 
 // WriteFile will write contents of the string to a file in
 // the artifacts directory. It will create the artifacts dir
 // if necessary.
 // Returns the full path (including the artifacts dir)
-func WriteFile(filename, contents string) (string, error) {
-	artifactDir := ads
-	_, err := createArtifactsDir(artifactDir)
-	if err != nil {
-		// Fatal does an os.Exit. If we can't create the artifacts directory,
-		// we can't continue.
-		log.Fatal(fmt.Errorf("could not create artifact path: %v", err))
-	}
+func WriteFile(filename string, contents io.Reader) (string, error) {
```

Comment:
We're no longer creating the artifacts directory on write. Is that intentional? If so, we need to update this comment.

Reply:
I'm reading `SafeWriteReader` which looks like it creates parent directories, so we may still be doing the same thing here. Just make sure I didn't misread that if you don't mind.

That said, it looks like this will throw an error if we try to overwrite a file:

https://github.com/spf13/afero/blob/v1.9.2/util.go#L84-L86

On a single iteration, I think this is fine. I wonder if subsequent iterations becomes more frustrating to deal with because it requires an `rm -rf artifacts/` before a re-execution.

Reply:
You are correct on all counts. 

As for the need for an rm -rf... I would actually prefer that be the case. I don't think we really want artifacts from different runs mixed up anyway. 

We could use WriteReader instead. It skips the Exists check, but keeps all of the other conveniences.

Reply:
Leave it until it becomes a problem. Probably best folks clean up artifacts in between runs anyway, as you indicate.